unittests/api_tests.cpp

```diff
@@ -404,55 +404,57 @@ BOOST_FIXTURE_TEST_CASE(action_receipt_tests, TESTER) { try {
 /*************************************************************************************
  * action_tests test case
  *************************************************************************************/
-BOOST_FIXTURE_TEST_CASE(action_tests, TESTER) { try {
-	produce_blocks(2);
-	create_account( N(testapi) );
-	create_account( N(acc1) );
-	create_account( N(acc2) );
-	create_account( N(acc3) );
-	create_account( N(acc4) );
-	produce_blocks(10);
-	set_code( N(testapi), contracts::test_api_wasm() );
-	produce_blocks(1);
+BOOST_AUTO_TEST_CASE(action_receipt) { try {
+   validating_tester chain( {}, {::eosio::chain::builtin_protocol_feature_t::stop_deferred_transactions} );
```

Comment:
The original code uses TESTER.  Can you make tester define the same constructors as validating_tester?

Reply:
Made matching constructors for class `tester`.